R/extract.R

```diff
@@ -5441,4 +5441,48 @@ extract.gnm <- function(model, include.aic = TRUE, include.bic = TRUE,
 }
 
 setMethod("extract", signature = className("gnm", "gnm"),
-          definition = extract.gnm)
\ No newline at end of file
+          definition = extract.gnm)
+
+# extention for panelAR objects (panelAR package)					 
```

Comment:
I noticed that you added all those recent extract functions to the end of the file. They all used to be in alphabetical order in the `extract.R` file. Can you rearrange that in alphabetical order when you get the chance?